python/cudf/cudf/tests/test_string.py

```diff
@@ -223,21 +223,14 @@ def test_string_numeric_astype(dtype):
     elif dtype.startswith("float"):
         data = [1.0, 2.0, 3.0, 4.0, 5.0]
     elif dtype.startswith("datetime64"):
-        data = [1000000000, 2000000000, 3000000000, 4000000000, 5000000000]
-    if dtype.startswith("datetime64"):
-        ps = pd.Series(data, dtype="datetime64[ns]")
-        gs = Series.from_pandas(ps)
-    else:
-        ps = pd.Series(data, dtype=dtype)
-        gs = Series(data, dtype=dtype)
+        # pandas rounds the output format based on the data
+        data = [1000000001, 2000000001, 3000000001, 4000000001, 5000000001]
+        if dtype != "datetime64[ns]":
+            pytest.xfail(reason=f"Pandas does not support {dtype}")
```

Comment:
skip instead of xfailing here or update test to use numpy instead of Pandas for this testing